### ISSUE TITLE
Buidling the frontend throws error

### ISSUE THREAD

**Issue by christian-vorhemus** [Microsoft/AzureSearch_JFK_Files#55]
When running
`npm run build:prod`

I receive
`ERROR in [at-loader] ./src/pages/search-page/components/item/item.component.tsx:41:7
    TS2326: Types of property 'component' are incompatible.
  Type '"img"' is not assignable to type 'ReactType<CardMediaProps>'.`

As a quick workaround I changed the CardMedia component in item.component.tsx to

```typescript
const ItemMediaThumbnail: React.StatelessComponent<ItemProps> = ({ item, onClick }) => {
  return (
    item.thumbnail ? 
    <CardMedia className={style.media}
      image={item.thumbnail}        
      title={item.title}
      onClick={handleOnClick({ item, onClick })}
    /> : null
  );
}
```

**Comment by AlexFiala** [comment]
I also experienced this issue.  Any reason this occurs?

**Comment by Careyjmac** [comment]
Solved by #56
